csrfguard/src/main/java/org/owasp/csrfguard/CsrfGuard.java

```diff
@@ -177,6 +182,7 @@ public LogicalSessionExtractor getLogicalSessionExtractor() {
      */
     public void initializeJavaScriptConfiguration() {
         config().initializeJavaScriptConfiguration();
+        initialiseJavaScriptConfigurationForNewConfigurationInstances = true;
```

Comment:
please prefix class fields with `this` for readability, and to be uniform with the rest of the code-base